app/controllers/npq_separation/admin/finance/statements/adjustments_controller.rb

```diff
@@ -2,19 +2,27 @@
 
 class NpqSeparation::Admin::Finance::Statements::AdjustmentsController < NpqSeparation::AdminController
   before_action :set_statement
+  before_action :find_adjustment, only: %i[edit update delete destroy]
+  before_action :set_show_all_adjustments, except: %i[create add_another]
 
   def new
     @adjustment = @statement.adjustments.new
+    @cancel_url = if @show_all_adjustments
+                    npq_separation_admin_finance_statement_adjustments_path(@statement, show_all_adjustments: @show_all_adjustments)
+                  else
+                    npq_separation_admin_finance_statement_path(@statement)
+                  end
   end
 
   def create
     @create_adjustment_form = Admin::Adjustments::CreateAdjustmentForm.new(
       adjustment_params.merge(created_adjustment_ids: session[:created_adjustment_ids], statement: @statement),
     )
 
-    if @create_adjustment_form.save_form
+    if @create_adjustment_form.save_adjustment
       session[:created_adjustment_ids] = @create_adjustment_form.created_adjustment_ids
-      redirect_to npq_separation_admin_finance_statement_adjustments_path(@statement)
+      show_all_adjustments = params[:adjustment][:show_all_adjustments] == "true"
```

Comment:
This line is also used below? Could it be moved to its own `show_all_adjustments?` method?

Reply:
not really - the two are different - different form names:
line 24: `show_all_adjustments = params[:adjustment][:show_all_adjustments] == "true"`
line 64: `show_all_adjustments = params[:add_another_form][:show_all_adjustments] == "true"`